libraries/botbuilder/src/skills/skillHandler.ts

```diff
@@ -118,9 +118,9 @@ export class SkillHandler extends ChannelServiceHandler {
      * This method allows you to reply to an activity.
      *
      * This is slightly different from sendToConversation().
-     * * sendToConversation(conversationId) - will append the activity to the end
+     * sendToConversation(conversationId) - will append the activity to the end
      * of the conversation according to the timestamp or semantics of the channel.
-     * * replyToActivity(conversationId,ActivityId) - adds the activity as a reply
+     * replyToActivity(conversationId,ActivityId) - adds the activity as a reply
```

Comment:
Same

Reply:
fixed

Reply:
The `-` should replace the `*` in both 2 lines